src/transformers/trainer.py

```diff
@@ -1297,7 +1297,8 @@ def evaluate(self, eval_dataset: Optional[Dataset] = None) -> Dict[str, float]:
                 :obj:`__len__` method.
 
         Returns:
-            A dictionary containing the evaluation loss and the potential metrics computed from the predictions.
+            A dictionary containing the evaluation loss and the potential metrics computed from the predictions. The
+            dictionary also contains the epoch number which comes from the training state. It can just be ignored.
```

Comment:
```suggestion
            dictionary also contains the epoch number which comes from the training state.
```
I wouldn't add the last sentence since it can be useful when the evaluation is done inside the training loop.